### ISSUE TITLE
[reqeust help] 请帮助我们提供 playbook 管理功能

### ISSUE THREAD

**Issue by Wine93** [opencurve/curveadm#343]
目前我们的 playbook 脚本仍存储在 CurveAdm 的[主仓库](https://github.com/opencurve/curveadm/tree/develop/playbook)中，用户在使用 playbook 时需要将对应的脚本拷贝到本地进行执行，我们希望你为我们提供一个 playbook 安装功能，可以快速将指定的 playbook 脚本集从远端安装到本地 CurveAdm 的主目录下：
```bash
curveadm playbook install memcache
```
此外，我们还希望你能帮我们改进 playbook 执行方式，给我们提供像 ansible 一样的模块选项，例如：

```
curveadm playbook -m memcache deploy
```

以上这个示例命令会选择用户已经安装的 memcache 模块，并执行 deploy 操作。

 HINTS
---

* 以上只是初步的设计方案，如果你有更好的建议，请回复该 issue 告诉我们
* 我们系统有一个集中仓库来管理用户的 playbook，要是能让用户自由提交就更好了，类似 oh-my-zsh 的插件
* 此外，我们可以为你提供 NOS 存储，用来存储 playbook 脚本

**Comment by hdbdn77** [comment]
这个问题是因为安装curveadm时，默认不会下载 playbook 脚本导致的吗

**Comment by Wine93** [comment]
> 这个问题是因为安装curveadm时，默认不会下载 playbook 脚本导致的吗

是的。可以在安装时将一些必要的 playbook 安装到主目录下，此外也需要提供管理命令，供已安装用户下载指定的 playbook，以及更新 playbook 版本等。

**Comment by hdbdn77** [comment]
好的，我想尝试一下，但我希望先完成Curve Autumn Code Camp的任务

**Comment by caoxianfei1** [comment]
> 好的，我想尝试一下，但我希望先完成Curve Autumn Code Camp的任务

好的
